images/cilium/download-hubble.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2017-2021 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# shellcheck source=hubble-version.sh
+source "${script_dir}/hubble-version.sh"
+
+for arch in amd64 arm64 ; do
```

Comment:
Why do we need to install both amd64 and arm64? If this is called from a Dockerfile there will be two different calls, one for each `TARGETARCH`.

Reply:
IIRC this step will only be executed once. Docker will cache the layer and reuse if for the other architectures.